### PR TITLE
fix(database-change): missing sql plan task execution result 

### PR DESCRIPTION
**Pull request by kiko-art**
#### What type of PR is this?
type-bug

#### What this PR does / why we need it:
sql plan task add schedule job, the value of task_result is obtained from the job_job.
#### Which issue(s) this PR fixes:

Fixes #

#### Additional documentation e.g., usage docs, etc.:

```docs

```

### CODE REVIEW

File: server/odc-service/src/main/java/com/oceanbase/odc/service/schedule/ScheduleTaskService.java [hunk 1]
```diff
@@ -291,6 +294,14 @@ public Page<ScheduleTaskListOverview> getConditionalScheduleTaskListResp(Pageabl
         Map<Long, Database> databaseMap = scheduleResponseMapperFactory.getDatabaseInfoByIds(databaseIds).stream()
                 .collect(Collectors.toMap(Database::getId, Function.identity()));
 
+        // get job result json
+        List<Long> jobIds = scheduleTaskPage.getContent().stream().map(ScheduleTask::getJobId).collect(
```
Comment (guowl3) [inline server/odc-service/src/main/java/com/oceanbase/odc/service/schedule/ScheduleTaskService.java:298]: need to check if jobId is null

File: server/odc-service/src/main/java/com/oceanbase/odc/service/task/executor/task/SqlPlanTask.java [hunk 2]
```diff
@@ -97,6 +100,17 @@ protected void doInit(JobContext context) {
                 PublishSqlPlanJobReq.class);
         this.connectionSession = generateSession();
         this.executor = connectionSession.getSyncJdbcExecutor(ConnectionSessionConstants.CONSOLE_DS_KEY);
+        long timeoutUs = TimeUnit.MILLISECONDS.toMicros(parameters.getTimeoutMillis());
+        if (timeoutUs < 0) {
+            throw new IllegalArgumentException(
```
Comment (yizhouxw) [inline server/odc-service/src/main/java/com/oceanbase/odc/service/task/executor/task/SqlPlanTask.java:105]: the error message is not correct, may simplify as 'Invalid timeout settings', and please use PreConditions  instead

File: server/odc-service/src/main/java/com/oceanbase/odc/service/schedule/ScheduleTaskService.java (hunk 1 above)
Comment (kiko-art) [reply server/odc-service/src/main/java/com/oceanbase/odc/service/schedule/ScheduleTaskService.java]: Have been modified

File: server/odc-service/src/main/java/com/oceanbase/odc/service/task/executor/task/SqlPlanTask.java (hunk 2 above)
Comment (kiko-art) [reply server/odc-service/src/main/java/com/oceanbase/odc/service/task/executor/task/SqlPlanTask.java:106]: Have been modified.

**Review by yizhouxw — APPROVED** [review]
lgtm

**Review by guowl3 — APPROVED** [review]
lgtm

**Review by yhilmare — APPROVED** [review]
lgtm

**Pull request merged**